fpr/migrations/0007_pronom_89.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+from fpr.models import Format, FormatVersion, IDRule
```

Comment:
We can't do this. I've found the only way to deal with this issue properly is updating django-autoslug to 1.9.3.